base/client.jl

```diff
@@ -366,7 +373,11 @@ function __atreplinit(repl)
 end
 _atreplinit(repl) = invokelatest(__atreplinit, repl)
 
+# The REPL stdlib hooks into Base using this Ref
+const REPLREF = Ref{Module}()
```

Comment:
I find the name `REPLREF` a bit confusing, this could suggest that it's a ref to a "REPL object" (not module), what about `REPLMOD` ? (not a big deal anyway).

Reply:
I'll change it to `REPL_MODULE_REF`.